Library/Formula/terraform.rb

```diff
@@ -145,14 +63,12 @@ def install
     end
 
     cd terrapath do
-      system "go", "test", "./..."
+      terraform_files = `go list ./... | grep -v /vendor/`.strip.gsub("\n", " ")
+      system "go test #{terraform_files}"
```

Comment:
now with the subshell invocation removed it should work.
rubys `system` method doesn't use a real shell when using the array notation, which is why I changed it to use the subshell.